docs/src/installation.md

```diff
@@ -29,7 +50,9 @@ Pkg.build("RCall")
 
 When `R_HOME` is set to `"*"`, RCall.jl will automatically install R for you using [Conda](https://github.com/JuliaPy/Conda.jl).
 
-Should you experience problems with any of these methods, please [open an issue](https://github.com/JuliaStats/RCall.jl/issues/new).
+!!! note "Installation time R installation is shared"
+
+    When the R installation is configured at installation time, the absolute path to the R installation is currently hard-coded into the RCall.jl package, which can be shared between projects. This may cause problems if you are using different R installations for different projects which end up using the same RCall.jl installation. In this case, please [use the Preferences system instead](#Customizing-the-R-installation-at-install-time) which keeps different copies of the compiled RCall for different R installations.
 
```

Comment:
It would be good to note explicitly again that the preferences system can be used post-hoc and doesn't require a re-build.

Reply:
Actually, you have to call `Pkg.build()` again to activate the preferences file if you've previously built using a global install, right?